pom.xml

```diff
@@ -122,7 +122,12 @@
 		    <groupId>org.apache.commons</groupId>
 		    <artifactId>commons-csv</artifactId>
 		    <version>1.5</version>
-			</dependency>			
+			</dependency>
+			<dependency>
+	      <groupId>com.google.guava</groupId>
```

Comment:
What's this dependency for? I just ran my version of the tests and I think they ran successfully. Is it possible adding this dependency is your problem?

Reply:
I added to get something working but it's not needed. I am going to remove it.